src/TLuaInterpreter.cpp

```diff
@@ -5138,12 +5138,38 @@ int TLuaInterpreter::echoUserWindow(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setAppStyleSheet
 int TLuaInterpreter::setAppStyleSheet(lua_State* L)
 {
-    if (lua_isstring(L, 1)) {
-        string stylesheet = lua_tostring(L, 1);
-        qApp->setStyleSheet(stylesheet.c_str());
+    QString styleSheet;
+    QString tag;
+    int s = 0;
+    int n = lua_gettop(L);
+    if (n) {
+        if (!lua_isstring(L, ++s)) {
+            lua_pushfstring(L, "setAppStyleSheet: bad argument #%d type (style sheet as string expected {may be omitted to remove existing stylesheet}, got %s!)", s, luaL_typename(L, s));
```

Comment:
Might it seem strange to suggest that the argument is optional without saying why - given the indicated purpose of the command?

On the other hand calling `setAppStyleSheet("")` is not much more typing than calling `setAppStyleSheet()` even if the latter does have a certain attraction to suggest setting *no* style-sheet...

Reply:
Okay, I'll take out that detail - so calling it with no arguments will zap an existing application style sheet (as will a `setAppStyleSheet("")` or even `setAppStyleSheet([[ ]])`!) but it will report that a string is expected if any argument that is not one is passed.

Reply:
Perfect!